sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/ClientCommon.cs

```diff
@@ -128,5 +128,15 @@ public static string GetFeedbackId(string locationHeader)
                 throw new ArgumentException(UnexpectedHeaderFormat);
             }
         }
+
+        internal static HookInfoPatch GetPatchModel(AlertingHook hook)
```

Comment:
I think it would make more sense if we moved this to `AlertingHook`, like you did with the alert and anomaly detection configurations.

Reply:
oops - I did, and just never deleted it from here.